src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,11 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+export const UserInfo = ({ user }) => {
+  if (user) {
+    return (
```

Comment:
Alternatively, you may add this  {**todo.user &&** UserInfo user={todo.user} /} in your TodoInfo block

Reply:
Agree, @kylyvnyk solution seems to be more correct according to the requirements:

> UserInfo takes a user and renders the details (**use it only if todo.user exists**).